openfl/experimental/runtime/local_runtime.py

```diff
@@ -97,22 +219,27 @@ def some_collaborator_task(self):
                                              collaborator tasks will run sequentially.
         """
         super().__init__()
-        if backend not in ["ray", "single_process"]:
+        if backend not in ["ray", "single_process", "ray_grouped"]:

```

Comment:
The changes you've added should become the `ray` backend. `ray` and `ray_grouped` will confuse users

Reply:
replaced ray with raygrouped logic